zerver/openapi/zulip.yaml

```diff
@@ -15733,6 +15733,10 @@ paths:
                           For organization administrators, this will include all private channels
                           in the organization.
 
+                          **Changes**: Before Zulip 11.0 (feature level 376), archived channels
```

Comment:
We do not mention anything about this data including archived channels other than in "Changes" entry. We do the same for subscription data.